plugins/additional/xml/additional.pythoncode

```diff
@@ -27,113 +27,113 @@ Python code generation written by
+    <template name="evt_connect_OnHtmlCellClicked">self.$name.Bind(wx.html.EVT_HTML_CELL_CLICKED, #handler)</template>
+    <template name="evt_connect_OnHtmlCellHover">self.$name.Bind(wx.html.EVT_HTML_CELL_HOVER, #handler)</template>
+    <template name="evt_connect_OnHtmlLinkClicked">self.$name.Bind(wx.html.EVT_HTML_LINK_CLICKED, #handler)</template>
   </templates>
 
   <templates class="wxRichTextCtrl">
-    <template name="construction">self.$name = #class( #wxparent $name, $id, $value, $pos, $size, $style #ifnotnull $window_style @{ |$window_style @} #ifnotnull $window_name @{, wx.DefaultValidator, $window_name @} )</template>
+    <template name="construction">self.$name = #class(#wxparent $name, $id, $value, $pos, $size, $style #ifnotnull $window_style @{ |$window_style @} #ifnotnull $window_name @{, wx.DefaultValidator, $window_name @})</template>
     <template name="include">import wx.richtext</template>
-    <template name="evt_connect_OnText">self.$name.Bind( wx.EVT_TEXT, #handler )</template>
-    <template name="evt_connect_OnTextEnter">self.$name.Bind( wx.EVT_TEXT_ENTER, #handler )</template>
-    <template name="evt_connect_OnTextURL">self.$name.Bind( wx.EVT_TEXT_URL, #handler )</template>
-    <template name="evt_connect_OnTextMaxLen">self.$name.Bind( wx.EVT_TEXT_MAXLEN, #handler )</template>ate>
-    <template name="evt_connect_OnRichTextCharacter">self.$name.Bind( wx.richtext.EVT_RICHTEXT_CHARACTER, #handler )</template>
-    <template name="evt_connect_OnRichTextDelete">self.$name.Bind( wx.richtext.EVT_RICHTEXT_DELETE, #handler )</template>
-    <template name="evt_connect_OnRichTextReturn">self.$name.Bind( wx.richtext.EVT_RICHTEXT_RETURN, #handler )</template>
-    <template name="evt_connect_OnRichTextStyleChanged">self.$name.Bind( wx.richtext.EVT_RICHTEXT_STYLE_CHANGED, #handler )</template>
-    <template name="evt_connect_OnRichTextStyleSheetChanged">self.$name.Bind( wx.richtext.EVT_RICHTEXT_STYLESHEET_CHANGED, #handler )</template>
-    <template name="evt_connect_OnRichTextStyleSheetReplacing">self.$name.Bind( wx.richtext.EVT_RICHTEXT_STYLESHEET_REPLACING, #handler )</template>
-    <template name="evt_connect_OnRichTextStyleSheetReplaced">self.$name.Bind( wx.richtext.EVT_RICHTEXT_STYLESHEET_REPLACED, #handler )</template>
-    <template name="evt_connect_OnRichTextContentInserted">self.$name.Bind( wx.richtext.EVT_RICHTEXT_CONTENT_INSERTED, #handler )</template>
-    <template name="evt_connect_OnRichTextContentDeleted">$self.$name.Bind( wx.richtext.EVT_RICHTEXT_CONTENT_DELETED, #handler )</template>
+    <template name="evt_connect_OnText">self.$name.Bind(wx.EVT_TEXT, #handler)</template>
+    <template name="evt_connect_OnTextEnter">self.$name.Bind(wx.EVT_TEXT_ENTER, #handler)</template>
+    <template name="evt_connect_OnTextURL">self.$name.Bind(wx.EVT_TEXT_URL, #handler)</template>
+    <template name="evt_connect_OnTextMaxLen">self.$name.Bind(wx.EVT_TEXT_MAXLEN, #handler)</template>ate>
```

Comment:
There is a syntax error at the end of the line, this was already present before the change, just marking it so it doesn't get lost.